cmd/frontend/internal/auth/accessrequest/handlers.go

```diff
@@ -0,0 +1,91 @@
+package accessrequest
+
+import (
+	"encoding/json"
+
+	"github.com/sourcegraph/log"
+
+	"net/http"
+
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/internal/auth/userpasswd"
+	sgactor "github.com/sourcegraph/sourcegraph/internal/actor"
```

Comment:
looks like an alias for actor can be skipped here
```suggestion
	"github.com/sourcegraph/sourcegraph/internal/actor"
```

Reply:
Thank you, updated!